### ISSUE TITLE
Return the session count to the leftmost column in the display.

### ISSUE THREAD

**Issue by tallthom** [danmarsden/moodle-mod_attendance#652]
In older versions of Attendance, you could see the session count in the leftmost column of the display. However, in the newest version of Attendance on 4.1 the column has gone away. This makes it difficult to count the number of sessions on the page if the list gets longer than 4 or 5 sessions.

Please provide a count of sessions on the page in some way. It could be a total number displayed on the top of the page, or it could be returning the session number to the left side.
<img width="1375" alt="Screenshot 2023-01-23 at 9 15 19" src="https://user-images.githubusercontent.com/1357329/213948295-2de3974a-026a-4852-8660-2925d5b2f2f8.png">

**Comment by danmarsden** [comment]
Thanks @tallthom - I'll keep it in mind - in the longer term I really want to replace this report with Moodle's new report builder api, but that api doesn't currently support bulk selection of items in the report yet (https://tracker.moodle.org/browse/MDL-72831)

**Comment by tallthom** [comment]
Thank you, @danmarsden -- I appreciate your quick response. Happy to have this report appear elsewhere or be generated elsewhere on the site as long as the new version of the report contains the total number somewhere (either line-by-line numbering, or a total at the bottom or top), the issue would be resolved for me. However, it seems like something that used to be there should be easily re-added back into the current report... just sayin' ;) Again, thanks for your considerations.

**Comment by danmarsden** [comment]
yep - thanks for the feedback. I have no intention of bringing back the left hand column with row numbers (no other core Moodle reports do this either) but adding back some form of record count to the page seems like a useful thing to do. I don't expect to make any further changes to the existing report until I change to using report builder api's but pull requests are always welcome.
